### PR TITLE
[refactor]: Modify comment model

### LINKED ISSUE #131: [FEATURE REQUEST] - Modify Comment Model
### Is your feature request related to a problem? Please describe.
Based on deliberations and suggestions from contributors, the comments model needs to be modified

### Additional context
None

### PR DESCRIPTION
**Pull request by dave-ok**
**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**

Fixes #131

- Removed unneeded user model field to leave user management to 3rd party applications
- Modified flags field, upvotes and downvotes fields into array of userIds

**What is the link to the issue on Github?**

[Issue #131](https://github.com/microapidev/comment-microapi/issues/131)

### CODE REVIEW

File: models/comments.js [hunk 1]
```diff
@@ -3,31 +3,44 @@ const Schema = mongoose.Schema;
...
+      // field used for emails, userId from the applcation used to identify its users
       type: String,
       required: true,
     },
-    commentOrigin: String,
-    isFlagged: { type: Boolean, default: false },
-    numOfFlags: {
-      type: Number,
-      default: 0,
+    commentBody: {
```
Comment (simply-alliv) [inline models/comments.js:17]: Can we omit the comment part?

File: models/comments.js [hunk 2]
```diff
@@ -3,31 +3,44 @@ const Schema = mongoose.Schema;
...
+      // field contains comment body
+      type: String,
+      required: true,
     },
-    replies: [{ type: Schema.Types.ObjectId, ref: "Replies" }],
+    commentOrigin: String, // field is free for use by applications as seen fit
```
Comment (simply-alliv) [inline models/comments.js:22]: Same here with the comment part

File: models/comments.js (hunk 1 above)
Comment (dave-ok) [reply models/comments.js]: Done

File: models/comments.js (hunk 2 above)
Comment (dave-ok) [reply models/comments.js]: Done

**Pull request merged**